app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
```

Comment:
### Optional
- [ ] It would be nice if we make our `update_posts_counter` private so that the method will only be available for the class